src/combinator.cpp

```diff
@@ -93,6 +201,15 @@ void Combinator::on_transformer_config(const metricq::json& config)
         // Register the combined metric as a new source metric
         auto& metric = (*this)[combined_name];
 
+        try
+        {
+            metric.metadata["displayExpression"] = displayExpression(combined_expression);
+        }
+        catch (std::runtime_error&)
+        {
+            Log::error("Failed to create the Display Expression");
```

Comment:
Please add context to the error message.

- what is the affected metric?
- what is the error message of the runtime error?